hazelcast/src/main/java/com/hazelcast/internal/partition/impl/MigrationStats.java

```diff
@@ -93,19 +144,21 @@ void incrementCompletedMigrations() {
         totalCompletedMigrations.incrementAndGet();
     }
 
-    void recordMigrationOperationTime(long time) {
-        elapsedMigrationOperationTime.addAndGet(time);
-        totalElapsedMigrationOperationTime.addAndGet(time);
+    private void calculateElapsed(final AtomicLong elapsedTime, final AtomicLong totalElapsedTime) {
+        elapsedTime.set(Duration.ofMillis(Clock.currentTimeMillis()).minusMillis(lastRepartitionTime.get()).toNanos());
+        totalElapsedTime.set(elapsedTime.get());
```

Comment:
If we're tracking everything based on wall-clock, there's not much value to separate counters for `elapsedTime` vs `totalElapsedTime`, especially when we're just setting the latter to the former.

Reply:
I agree that the `total` ones don't seem to be adding any value, but they were already producing identical results before (i.e. `totalElapsedMigrationTime`=`elapsedMigrationTime`).

Reply:
> The difference between the "total" and "non-total" variables is that "non-total" is a tally over the current repartitioning round, whereas "total" is across all repartitioning rounds - see `MigrationStats#markNewRepartition`. Does your tests conduct multiple rounds of repartitioning?

Ah - I missed this completely, several times - I hadn't realised the values were different, so that's something I've broken I'll need to fix.